hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipUpdateTest.java

```diff
@@ -112,6 +113,11 @@ public void init() {
         factory = createHazelcastInstanceFactory();
     }
 
+    @After
+    public void cleanUp() {
```

Comment:
Have you observed an issue with instances not shutting down in this test class? I expect `HazelcastTestSupport`'s `@After` method takes care of that.

Reply:
I have not, this is just some leftover from when I was originally homing my test in this class, thanks for pointing it out - removed in this commit https://github.com/hazelcast/hazelcast/pull/25114/commits/68dc8ba53785e67f8c2a01c92799529e9ed22c96 👍